### DISCUSSION TITLE
What is the connector at the top of the hotend breakout board called and where can I get some.

### DISCUSSION THREAD

**Discussion by KiwiHuman** [bassamanator/everything-sovol-sv06#34] [Q&A]
I am trying to find out what the name of [this](https://i.imgur.com/2tJ7BZn.png) connector (P1) is so I can Buy/make an extension cable and run my SV06 in an enclosure with the electronics outside.

**Comment by bassamanator** [comment]
You can find some info on the print head cable from [sovol repo](https://github.com/Sovol3d/SV06-Fully-Open-Source/blob/main/SV06%20Mechanical%20PDF/Adapter%20board-motherboard%20cable.pdf) and from [here](https://www.reddit.com/r/Sovol/comments/11o991h/sv06_replacement_print_head_ribbon_cable/).

I have yet to come across this cable in the wild. I would contact sovol to see if they're willing to sell you a *longer* version from their stock (really this option should be listed on their website, it's another part that they can sell).

Keep us in the loop.
